@here/harp-omv-datasource/lib/OmvDecodedTileEmitter.ts

```diff
@@ -355,28 +304,37 @@ export class OmvDecodedTileEmitter implements IOmvEmitter {
             const techniqueName = technique.name;
 
             if (techniqueName === "line") {
-                applyLineTechnique(
+                this.applyLineTechnique(
```

Comment:
This triple if statement could be easily modified to be a single one where the inputs can be computed by simple ternary operators (``const d = a ? b: c;``). Would add some readibility to the code.